src/modlistviewactions.h

```diff
@@ -31,9 +31,10 @@ class ModListViewActions : public QObject
     PluginListView* pluginView,
     QObject* nxmReceiver);
 
-  // install the mod from the given archive
+  // install the mod from the given archive before "above" the mod with
```

Comment:
Comment is outdated now that it also considers separators and sorting order

Reply:
Fixed.